packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -1683,9 +1685,7 @@ class InteractiveGraph extends React.Component<Props, State> {
     focus: () => void = $.noop;
 
     render(): React.ReactNode {
-        const box = getInteractiveBoxFromSizeClass(
-            this.props.containerSizeClass,
-        );
+        const box = getInteractiveBoxFromSizeClass(containerSizeClass.SMALL);
```

Comment:
Hard coding this to small for all the reasons mentioned above. It doesn't look like there's any way for the content editor to change it to anything else either.